internal/logging/handlers/handler.go

```diff
@@ -52,5 +67,205 @@ func BuildOptions(ctx context.Context, k8s client.Client, mcAddon *addonapiv1alp
 	}
 	opts.Secrets = secrets
 
+	configMaps, err := addon.GetConfigMaps(ctx, k8s, clf.Namespace, mcAddon.Namespace, configmapNames)
+	if err != nil {
+		return opts, err
+	}
+	opts.ConfigMaps = configMaps
+
 	return opts, nil
 }
+
+func getOutputResourcesNames(output loggingv1.OutputSpec) ([]string, []string, error) {
+	extractedSecretsNames := map[string]struct{}{}
+	extractedConfigMapNames := map[string]struct{}{}
+
+	getSecretsFromTokenAuthentication := func(secretNames map[string]struct{}, token *loggingv1.BearerToken) {
+		switch token.From {
+		case loggingv1.BearerTokenFromSecret:
+			secretNames[token.Secret.Name] = struct{}{}
+		case loggingv1.BearerTokenFromServiceAccount:
+			// In this authentication method MCOA should't do anything since
+			// the token is associated with the SA.
+			// Maybe we should instead create the necessary RBAC for the SA?
+		}
```

Comment:
I believe this is an interesting case to explore after the release in form of a `ManagedServiceAccount`: https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html-single/clusters/index#managed-serviceaccount-addon

Reply:
I'm not seeing the benefit of using `ManagedServiceAccont` over us doing the creation of the SA via the Helm chart 🤔